src/scripts/components/header/SearchInput.js

```diff
@@ -1,69 +1,85 @@
 import React from 'react';
-import {navigateAction} from 'fluxible-router';
+import {
+    navigateAction
+}
+from 'fluxible-router';
 import classNames from 'classnames';
 
 class SearchInput extends React.Component {
-  constructor(props, context) {
-    super(props, context);
-  }
+    constructor(props, context) {
+        super(props, context);
+    }
 
-  search(e) {
-    if (e.key === 'Enter' || e.keyCode === 13 || e.type === 'click') {
-      let searching = React.findDOMNode(this).querySelector('input').value,
-        ayah, pattern, surah;
+    search(e) {
+        if (e.key === 'Enter' || e.keyCode === 13 || e.type === 'click') {
+            let inputEl = React.findDOMNode(this).querySelector('input'),
+                searching = inputEl.value.trim(),
+                ayah, pattern, surah;
 
-      const shortcutSearch = /\d[\.,\:,\,,\\,//]/g;
-      const splitSearch = /[\.,\:,\,,\\,//]/g;
+            // prevent search function while search input field is empty
+            if (searching === '') {
```

Comment:
@mmahalwy This is the #143 fix. I hope you don't mind formating cleanup. Do we have a standard for formatting files? Suggetions?